docker-compose-google-pubsub.yml

```diff
@@ -3,25 +3,25 @@ version: '3'
 services:
 
   herbie-app:
-    container_name: 'herbie-app'
+    container_name: 'herbie_core-app'
```

Comment:
is there a reason for renaming herbie to herbie_core?

Reply:
I created demo-project in another repository:
https://github.com/herbie/sandbox/pull/1/files
There you will find the herbie and the herbieapp packages, I had to rename it here because otherwise the name will collide with those packages.

Reply:
ah ok, the name seems to be a bit confusing as with core I would assume this package/app is a part of another application

Reply:
Basically that is the case. This docker-compose-file is just for testing purpose.